### ISSUE TITLE
Referencing unknown named type in Attributes directly should give error

### ISSUE THREAD

**Issue by pksunkara** [apiaryio/snowcrash#354]
The following blueprint gives an error while parsing

```apib
# GET /
+ Response 200
    + Attributes
        + data (A)
```

but the following blueprint doesn't give an error.

```apib
# GET /
+ Response 200
    + Attributes (A)
```

The expected result is that both the blueprints should give an error while parsing.

**Comment by kylef** [comment]
This has been fixed already (at least in Drafter), since all semantical checks are done in Drafter I'm gonna close this issue.

```shell
$ cat x.apib 
# GET /
+ Response 200
    + Attributes (A)


$ ./bin/drafter x.apib -l

OK.
warning: (8)  unable to find the symbol `A` in the list of named types :29:15
```

**Comment by pksunkara** [comment]
Not a warning. It should be an error.

**Comment by pksunkara** [comment]
Which is why I opened the issue If I remember.

**Comment by kylef** [comment]
Good point, re-opened.

**Comment by pksunkara** [comment]
Fixed by #400
